server/server.go

```diff
@@ -7,20 +7,25 @@ import (
 	"io"
 	"log"
 	"net/http"
+	"sync"
 )
 
 // Parses the client request and uploads the file.
 func upload(w http.ResponseWriter, r *http.Request) {
 	fmt.Println("File Upload Endpoint Hit")
 
 	// Give a max limit for file upload.
-	r.ParseMultipartForm(10 << 20)
+	err := r.ParseMultipartForm(10 << 20)
```

Comment:
What's the reason for limiting the file sizes to within 10MB?
At the very least this should be mentioned in the README for now.